common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
You changed the CoA of Darkspear, but not of the Jin dynasty.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122809845-b029c700-d2df-11eb-9fe9-4908ea2bd311.png)

</details>